dags/analyzer_helper/discord/discord_load_transformed_data.py

```diff
@@ -0,0 +1,16 @@
+from dags.analyzer_helper.discord.load_transformed_data_base import LoadTransformedDataBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordLoadTransformedData(LoadTransformedDataBase):
+    def __init__(self, platform_id: str):
+        super().__init__(platform_id)
+        self.client = MongoSingleton.get_instance().client
+        self.db = self.client[self.get_platform_id()]
+        self.collection = self.db['rawmemberactivities']
+
+    def load(self, processed_data: list[dict], recompute: bool = False):
+        # Probably too aggressive, we should define another DAG for that
+        if recompute:
+            self.collection.delete_many({})
```

Comment:
Please add a log in the `if recompute` condition something like this "recompute is True, deleting the previous day"

For your comment on being too aggressive, if we wanted to pass data to a task on a DAG, it's not a good practice. The reason is Apache Airflow supports passing data from a task to another (XCom) but it has a limit and it is not recommended to pass large data from a task to another ( [doc link](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/xcoms.html#:~:text=An%20XCom%20is%20identified%20by%20a%20key%20(essentially%20its%20name)%2C%20as%20well%20as%20the%20task_id%20and%20dag_id%20it%20came%20from.%20They%20can%20have%20any%20(serializable)%20value%2C%20but%20they%20are%20only%20designed%20for%20small%20amounts%20of%20data%3B%20do%20not%20use%20them%20to%20pass%20around%20large%20values%2C%20like%20dataframes.) )

I assumed by DAG a task of a DAG but did you mean something else?

Reply:
> Please add a log in the if recompute condition something like this "recompute is True, deleting the previous day"

@polux0 Could you add the logs for deletion?